src/hotspot/share/runtime/synchronizer.hpp

```diff
@@ -36,6 +36,34 @@ class LogStream;
 class ObjectMonitor;
 class ThreadsList;
 
+class MonitorList {
```

Comment:
I guess we can't hide MonitorList and MonitorList::Iterator
in synchronizer.cpp as implementation details any more.